Core/HLE/sceUsbCam.cpp

```diff
@@ -116,7 +116,6 @@ static int sceUsbCamReadVideoFrame(u32 bufAddr, u32 size) {
 }
 
 static int sceUsbCamPollReadVideoFrameEnd() {
-	INFO_LOG(HLE, "UNIMPL sceUsbCamPollReadVideoFrameEnd: %d", nextVideoFrame);
```

Comment:
I'm not sure how interesting the returned value here is, but if it might be interesting for debugging, might want to change it to VERBOSE_LOG or DEBUG_LOG instead of deleting the line.

Reply:
(or use hleLogSuccessVerboseI if you want the arguments and return value logged.)

-[Unknown]